ui/apps/platform/cypress/helpers/tableHelpers.js

```diff
@@ -24,3 +24,17 @@ export function editIntegration(name) {
         `tr:contains('${name}') td.pf-v5-c-table__action button:contains("Edit Integration")`
     ).click();
 }
+
+export function queryTableHeader(headerName) {
+    return cy.get(`th`).contains(new RegExp(`^${headerName}$`, 'i'));
```

Comment:
While reading the tests, I noticed Title Case like **Affected Nodes** and **First Discovered** when user interface has Sentence case. What are pro and con for case sensitive match?